lib/libzpool/Makefile.am

```diff
@@ -190,7 +225,8 @@ LUA_C = \
 nodist_libzpool_la_SOURCES = \
 	$(USER_C) \
 	$(KERNEL_C) \
-	$(LUA_C)
+	$(LUA_C) \
+	$(ZSTD_C)
```

Comment:
Allan does ZSTD_C before LUA_C here, don't know if it has any functional effect (guess not) but it looks a bit cleaner...

Reply:
that does not make any difference here